### PR TITLE
Refactor HomeOwnersTable 

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* 
- bcgov/entity#16795

*Description of changes:*
- Refactor HomeOwnersTable (to use v-simple-table) and related components
- Update unit tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1390-xw9p0grk.web.app

File: ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue
Comment (orelbn) [inline ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue]: There seems to be some spacing adjustments that needs to be made for the edit button of owners in groups.
It's strange when I have my browser console open the spacing is fine, but when I just have my window on full screen its slightly off.

PR link:
image


Dev:
image

File: ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue
Comment (orelbn) [inline ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue]: The closing div here does not line up with the opening div.

File: ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue
Comment (orelbn) [inline ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue]: Can you double check the spacing here please.

File: ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue
Comment (orelbn) [inline ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue]: Can this be a template instead of div?

File: ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue
Comment (orelbn) [inline ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue]: Since there is no styling applied to this div, can it be a template?

**Review by orelbn — COMMENTED** [review]
Awesome job!  A couple styling and spacing things to look at, and then ill be good to approve.

File: ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue]: Minor but new lines for the cell properties and '>' tag.

**Review by cameron-eyds — APPROVED** [review]
Some minor comments to resolve but looks good to me. 
Hopefully we can get an 'all-hands' approach to QA'ing all these table updates. 
I'm sure there will be some quirks.

**Comment by dimak1** [comment]
@cameron-eyds @orelbn I formatted the code with the latest commit. I will hold off on the other small fixes until we are ready to merge/test and move on to work on Sprint Goals tasks instead.

**Review by doug-lovett — APPROVED** [review]
Looks good.

**Review by orelbn — APPROVED** [review]
Thanks for formatting! Looks good

**Comment by dimak1** [comment]
Update the table and fixed the styling.

Screenshot 2023-06-20 at 5 56 21 PM
Screenshot 2023-06-20 at 5 56 12 PM
Screenshot 2023-06-20 at 6 28 54 PM
Screenshot 2023-06-20 at 6 27 37 PM
Screenshot 2023-06-20 at 5 57 09 PM

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1390-xw9p0grk.web.app

**Review by cameron-eyds — APPROVED** [review]
looks good!

**Review by doug-lovett — APPROVED** [review]
Looks great.

**Review by orelbn — APPROVED** [review]
There is a small difference in the group headers between the DEV and PR link, but I don't think it matters much. So feel free to merge and move on to other tickets, or if you feel like addressing it go ahead.

DEV:
image

PR LINK:
image

**Pull request merged**
